### ISSUE TITLE
Error when I 'make runtest'

### ISSUE THREAD

**Issue by YuGongCharley** [weiliu89/caffe#970]
I don't know how to fix it , HELP！！
这个问题我怎么都解决不了，又要赶进度，求助！
### Issue summary
*** Aborted at 1548777631 (unix time) try "date -d @1548777631" if you are using GNU date ***
PC: @     0x7f6cda877428 gsignal
*** SIGABRT (@0x3e80000174a) received by PID 5962 (TID 0x7f6cdd49db00) from PID 5962; stack trace: ***
    @     0x7f6cda8774b0 (unknown)
    @     0x7f6cda877428 gsignal
    @     0x7f6cda87902a abort
    @     0x7f6cda8b97ea (unknown)
    @     0x7f6cda8c237a (unknown)
    @     0x7f6cda8c653c cfree
    @     0x7f6cdbbd08af google::protobuf::internal::DestroyDefaultRepeatedFields()
    @     0x7f6cdbbcfb3b google::protobuf::ShutdownProtobufLibrary()
    @     0x7f6ca43da3b9 (unknown)
    @     0x7f6cdd377de7 (unknown)
    @     0x7f6cda87bff8 (unknown)
    @     0x7f6cda87c045 exit
    @     0x7f6cda862837 __libc_start_main
    @           0x4080b9 _start
    @                0x0 (unknown)
Makefile:526: recipe for target 'runtest' failed
make: *** [runtest] Aborted (core dumped)

**Comment by chirs123456** [comment]
hi, l have the same questions. Did you solve it?

**Comment by YuGongCharley** [comment]
@chirs123456 
yes, I commented the "USE_CUDNN := 1" like "# USE_CUDNN := 1" in "Makefile.config", and it worked, my gpu is 750, maybe it can't work with cudnn

**Comment by PiseyYou** [comment]
> @chirs123456
> yes, I commented the "USE_CUDNN := 1" like "# USE_CUDNN := 1" in "Makefile.config", and it worked, my gpu is 750, maybe it can't work with cudnn

I did as this, but it doesn't work.

**Comment by sunian6183** [comment]
> > @chirs123456
> > yes, I commented the "USE_CUDNN := 1" like "# USE_CUDNN := 1" in "Makefile.config", and it worked, my gpu is 750, maybe it can't work with cudnn
> 
> I did as this, but it doesn't work.

I did this ,it doesn't work too.Did you solve it?

**Comment by celljy** [comment]
解决了吗，我import caffe成功了，但是make runtest fail.请问你是如何解决的
